pkg/util/constants/constants.go

```diff
@@ -30,4 +30,7 @@ const (
 
 	EnvOperatorPodName      = "MY_POD_NAME"
 	EnvOperatorPodNamespace = "MY_POD_NAMESPACE"
+
+	AnnotationScope       = "etcd.database.coreos.com/scope"
```

Comment:
Can you also move these const into k8sutil:
https://github.com/coreos/etcd-operator/blob/master/pkg/util/k8sutil/k8sutil.go#L46

We want to get rid of this file.

Reply:
remove `AnnotationScope`?